jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java

```diff
@@ -32,7 +32,7 @@
  */
 public class MonitorManager<T extends BaseMonitor> {
     /** The id used to distinguish monitors of different classloaders. */
-    private static String id;
+    private String id;
```

Comment:
MonitorManager stores a unique id of the parent class loader. This is to distinguish MonitorManagers in war style applications in case the MonitorManager is loaded outside of war scope.